duckscript_sdk/src/sdk/std/process/exec/mod_test.rs

```diff
@@ -14,28 +16,35 @@ fn run_no_args() {
 
 #[test]
 fn run_no_output() {
-    test::run_script_and_validate(vec![create("")], "exec echo test", CommandValidation::None);
+    let script = format!("exec {} \"echo test\"", COMMAND_RUNNER);
+    test::run_script_and_validate(vec![create("")], &script, CommandValidation::None);
 }
 
 #[test]
 fn run_no_output_with_fail_on_error_valid() {
+    let script = format!("exec --fail-on-error {} \"echo test\"", COMMAND_RUNNER);
+
     test::run_script_and_validate(
         vec![create("")],
-        "exec --fail-on-error echo test",
+        &script,
         CommandValidation::None,
     );
 }
 
 #[test]
 fn run_no_output_with_fail_on_error_invalid() {
-    test::run_script_and_error(vec![create("")], "exec --fail-on-error badcommand", "");
+    let script = format!("exec --fail-on-error {} badcommand", COMMAND_RUNNER);
+
+    test::run_script_and_error(vec![create("")], &script, "");
 }
 
 #[test]
 fn run_with_output() {
+    let script = format!("out = exec --fail-on-error {} \"echo 1 2 3\"", COMMAND_RUNNER);
```

Comment:
actually this is not the original command :)
we don't use fail on error here

Reply:
This is my fault. I spent a lot of time trying to pass the parameters to rust and  pasas to 'cmd/sh'. When I finally passed all the tests, I submitted it happily, forgot to modify it.